datalake/src/main/java/com/sequenceiq/datalake/flow/delete/event/StorageConsumptionCollectionUnschedulingSuccessEvent.java

```diff
@@ -0,0 +1,30 @@
+package com.sequenceiq.datalake.flow.delete.event;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.sequenceiq.datalake.flow.SdxEvent;
+import com.sequenceiq.datalake.flow.delete.SdxDeleteEvent;
+
+public class StorageConsumptionCollectionUnschedulingSuccessEvent extends SdxEvent {
+
+    private final boolean forced;
+
+    @JsonCreator
+    public StorageConsumptionCollectionUnschedulingSuccessEvent(
+            @JsonProperty("resourceId") Long sdxId,
+            @JsonProperty("userId") String userId,
+            @JsonProperty("forced") boolean forced) {
+        super(sdxId, userId);
+        this.forced = forced;
+    }
+
+    @Override
+    public String selector() {
+        return SdxDeleteEvent.STORAGE_CONSUMPTION_COLLECTION_UNSCHEDULING_SUCCESS_EVENT.event();
```

Comment:
you can use EventSelectorUtil

Reply:
The overriding impl has been removed like above.